staging/cse/windows/azurecnifunc.ps1

```diff
@@ -110,7 +110,35 @@ function Set-AzureCNIConfig
             $configJson.plugins.AdditionalArgs[0].Value.ExceptionList = $processedExceptions
         }
         else {
-            $configJson.plugins.AdditionalArgs[0].Value.ExceptionList = $exceptionAddresses
+            if ($IsDualStackEnabled) {

```

Comment:
Does IsDualStackEnabled only support WS2022? The else in line 112 is for `if ($osBuildNumber -le 17763)`.

Reply:
yes, there is no plan for IPv6 support to come to WS2019